### PR TITLE
Android Studio Projects

### PR DESCRIPTION
**Pull request by anshulkhattar**

### CODE REVIEW

File: AndroidProjects.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: Beginner-Level-AndroidStudio-Projects
+desc: this repository consists of some basic projects of android studio 
+site: https://github.com/anshulkhattar/Beginner-Level-Android-Studio-Apps
+tags:
+-Java
+-Android Studio
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/anshulkhattar/Beginner-Level-Android-Studio-Apps/issues
```
Comment (ritwik12) [inline AndroidProjects.yml]: @anshulkhattar Please change this to https://github.com/anshulkhattar/Beginner-Level-Android-Studio-Apps/labels/up-for-grabs

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: AndroidProjects.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: Beginner-Level-AndroidStudio-Projects
+desc: this repository consists of some basic projects of android studio
```
Comment (ritwik12) [inline AndroidProjects.yml:2]: @anshulkhattar Could this be improved to make it more clear?

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: AndroidProjects.yml (hunk 2 above)
Comment (anshulkhattar) [reply AndroidProjects.yml:2]: Made the desired changes, check it out.

File: AndroidProjects.yml [hunk 3]
```diff
@@ -1,9 +1,9 @@
-name: Beginner-Level-AndroidStudio-Projects
-desc: this repository consists of some basic projects of android studio 
-site: https://github.com/anshulkhattar/Beginner-Level-Android-Studio-Apps
```
Comment (ritwik12) [inline AndroidProjects.yml]: @anshulkhattar This was fine. Please correct.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: AndroidProjects.yml (hunk 3 above)
Comment (anshulkhattar) [reply AndroidProjects.yml]: Done, please check.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@anshulkhattar Thanks!!

**Comment by ritwik12** [comment]
@anshulkhattar Please correct path of file. and also name the file same as the name of your project in .yml

**Comment by ritwik12** [comment]
I reverted it

**Pull request merged**